aspnetcore/grpc/troubleshoot.md

```diff
@@ -170,6 +170,9 @@ The WPF application can use the gRPC generated types from the new class library
 
 ## Calling gRPC services hosted in a sub-directory
 
+> [!WARNING]
+> Many third-party gRPC tools don't support services hosted in subdomains. Consider finding a way to host gRPC as the root directory.
```

Comment:
```suggestion
> Many third-party gRPC tools don't support services hosted in subdirectories. Consider finding a way to host gRPC as the root directory.
```
> Many third-party gRPC tools don't support services hosted in subdomains.

subdomains are supported. It's services under a subdirectory that aren't.

![image](https://user-images.githubusercontent.com/303201/193608819-9ad9b1d3-624d-4f53-aa7f-1b4cc41d289d.png)